app/services/hyrax/fixity_status_service.rb

```diff
@@ -0,0 +1,86 @@
+module Hyrax
+  # Creates fixity status messages to display to user, for a fileset, or
+  # a specific version/file within it. Determines status by looking up
+  # existing ChecksumAuditLog objects, does not actually do a check itself.
+  # See FileSetFixityCheckService and ChecksumAuditLog for actually performing
+  # checks and recording as ChecksumAuditLog objects.
+  class FixityStatusService
+    include ActionView::Helpers::TagHelper
```

Comment:
I don't really understand the hydra architecture here.  I think I went with service object simply because it was before -- I separated out from one service object to another. And maybe I misunderstood a comment you made in slack that I thought was recommending extracting this to it's own service objects. 

Happy to change it to a 'presenter' object if you think that's better. I don't really understand the hydra 'presenter' architecture, can you share an analogous presenter object I can use as a guideline?  Can this still be it's _own_ presenter object that does nothing but status message, does that fit into the architecture?  I worry that trying to shoehorn this into some _existing_ presenter object that also does other stuff would be a bigger harm to backwards compat to people who have overridden that presenter object (as is encouraged I think?). 

While _most_ of the public API effected in this PR I think was not used by anyone, "display a fixity status message" of course is used in every sufia/hyrax app, and I tried to keep the code path to _that_ as backwards compat as possible (even leaving the cover method in FileSetFixityCheckService, for those who have presenters calling it).